views/base.html.twig

```diff
@@ -20,7 +20,7 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
-  <link rel="shortcut icon" href="{{ asset('theme/img/favicon.ico') }}">
+  <link rel="shortcut icon" href="/public/img/icons/favicon.ico" />
```

Comment:
we generally had problems clearing cache and deploying assets (it is not automated so needs to be done manually. Not everybody n the team - ie. @aleksandarjelicic - has ssh access to the server)